examples/lighting-app/nrfconnect/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ get_filename_component(NRFCONNECT_COMMON ${CHIP_ROOT}/examples/platform/nrfconne
 get_filename_component(LIGHTING_COMMON ${CHIP_ROOT}/examples/lighting-app/lighting-common REALPATH)
 
 set(CONF_FILE ${CHIP_ROOT}/config/nrfconnect/sample-defaults.conf prj.conf)
+set(CHIP_OPENTHREAD_CONFIG ${NRFCONNECT_COMMON}/project_include/OpenThreadConfig.h)
```

Comment:
I'm fine with this naming, but some other proposition:
`project_include/OpenThreadConfig.h` -> `config/openthread-project-core-config.h` (not sure about `project` part).

Reply:
Yeah, that would follow OpenThread naming convention, but in CHIP all examples seem to use `OpenThreadConfig.h`, so I decided to stay consistent within CHIP.